status-im/status-react
# [fix 3821] fix update-price to only update visible tokens

fix #3821

To test:

check if wallet works properly, check on testnet with STT and ETH if prices are displayed properly, checks if balance updates properly (compared to develop)

status: ready
